proposals/001-removing-sycl.rst

```diff
@@ -0,0 +1,84 @@
+==========================
+001: Removing SYCL Chapter
+==========================
+
+Remove the SYCL chapter from the oneAPI specification.
+
+See also:
+
+* `WG discussion`_
+
+.. _`WG discussion`: https://github.com/uxlfoundation/spec-working-group/blob/main/meetings/notes.rst#notes
+
+Background
+==========
+
+The SYCL chapter no longer serves a useful purpose and is causing some confusion
+about UXL's role in SYCL. The original motivation for the chapter was that DPC++
+was considered part of oneapi specification, and the chapter specifies the API
+that DPC++ supports, which is SYCL 2020 + extensions. With the adoption of SYCL
+2020, the need for extensions is reduced. DPC++ is not part of UXL. The hope is
+that the SYCL support in DPC++ will be upstreamed in LLVM, and that would be the
+home for specification of extensions, not the oneAPI specification.
+
+After the initial WG discussion, I followed up with Alexey & Greg.
+
+There are several issues:
+
+1. Policy/Documentation about use of SYCL vendor extensions in interfaces that
+   are part of oneapi spec
+2. Policy/Documentation about use of SYCL vendor extensions in UXL open source
+   projects.
+3. Documentation of use of ``std::`` from SYCL kernels
+
+Use of SYCL vendor extensions in interfaces
+===========================================
+
+I searched the header files by cloning all the UXL projects and the sources of
+the spec::
+
+    grep -n -r --include=\*.{h,hpp,rst} -E 'sycl::ext|ext_intel|ext_oneapi' .
+
+After review, I don't believe that extensions are currently part of the API. The
+closest to that is when a type defined in sycl::ext is made available in a
+library namespace::
+
+    namespace oneapi {
+    namespace mkl {
+
+    #ifndef __HIPSYCL__
+    using bfloat16 = sycl::ext::oneapi::bfloat16;
+    #endif
+
+
+I believe this was done to make ``oneapi::mkl::bfloat16`` available.
+
+Recommendation
+    Oneapi spec components should not use vendor extensions in interfaces. Either
+    wrap the type (example above) or work with SYCL WG to make it part of SYCL
+    spec or as a KHR extension. If there is a compelling reason to use a vendor
```

Comment:
KHR is undefined at this point. I see it's defined as the identifier for supra-vendor organization extensions in https://registry.khronos.org/EGL/specs/eglspec.1.5.pdf. Will most readers be familiar with it through background with Khronos, or is a brief definition needed?

Reply:
I updated the explanation. @gmlueck: this was based on our conversation so please review